packages/generator-electrode/generators/webapp/templates/server/conditions/machine-info.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
```

Comment:
should not copy these files unless user answers yes to `Disable server side rendering based on high load?`.  Consider moving these files to a separate module.